code/modules/mob/living/brain/posibrain.dm

```diff
@@ -222,3 +222,64 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 
 /obj/item/mmi/posibrain/display/is_occupied()
 	return TRUE
+
+/// Posibrains but spherical. They can roll around and you can kick them
+/obj/item/mmi/posibrain/sphere
+	name = "positronic sphere"
+	desc = "Recent developments on cost-cutting measures have allowed us to cut positronic brain cubes into twice-as-cheap spheres. \
+	Unfortunately, it also allows them to move around the lab via rolling maneuvers."
+	icon_state = "spheribrain"
+	base_icon_state = "spheribrain"
+	immobilize = FALSE
+	/// Delay between movements
+	var/move_delay = 0.5 SECONDS
+	/// when can we move again?
+	var/can_move
+
+/obj/item/mmi/posibrain/sphere/Initialize(mapload, autoping)
+	. = ..()
+
+	var/matrix/matrix = matrix()
+	transform = matrix.Scale(0.8, 0.8)
+
+	brainmob.remove_traits(list(TRAIT_IMMOBILIZED, TRAIT_HANDS_BLOCKED), BRAIN_UNAIDED)
+
+/obj/item/mmi/posibrain/sphere/relaymove(mob/living/user, direction)
+	if(isspaceturf(loc) || !direction || mecha)
+		return
+
+	if(can_move >= world.time)
+		return
+	can_move = world.time + move_delay
+
+	// ESCAPE PRISON
+	if(ismovable(loc) && prob(25))
+		var/obj/item/item = pick(loc.contents)
+		if(istype(loc, /obj/item/storage))
+			item.forceMove(get_turf(src)) //throw stuff out of the inventory till we free ourselves!
+			playsound(src, SFX_RUSTLE, 30, TRUE)
```

Comment:
Maybe a return after the movable check? We don't need to fall back to then checking if we are on a turf